Swift 3

Drewpom opened the pull request:
* Restructured project to match most Cocoapods and their generator script
 * Updated main files and pod to Swift 3
 * Upgraded all dependencies

frenetisch-applaudierend commented:
Hey Drew, thanks for helping out!

A few points:

By restructured to match most CocoaPods I assume you mean gitignoring the Pods directory. This is something I don't do on purpose (for safety and convenience when working on multiple branches with possibly different pods). If you could roll back that part, it would be appreciated.

I'm not entirely sure about the function of the Example folder, as it doesn't show anything of the function of this library. Did I miss something, or is it the intention to serve as a placeholder for a future sample app?

Thank you for taking your time to contribute!

Drewpom commented:
Cool, I just added back the pods directory.

As far as the example folder, I meant to add one but forgot. Just added a simple example there. It's not really the best example use-case, but I think it shows the basics. Let me know if there's anything else.

frenetisch-applaudierend commented:
Looks good, thanks
